venv/derivative_of_fractions.py

```diff
@@ -0,0 +1,38 @@
+#СЛОЖНОЕ И БЕСПОЛЕЗНОЕ ЗАДАНИЕ.ЕСТЬ ВТРОЕННЫЙ МЕТОД.НО ВСЕ РАБОТАЕТ АНАЛОГИЧНО FRACTIONS
+
+import math
+frac_tion1 = input("Введите первую дробь:").split("/")
+numerator1 = int(frac_tion1[0])#первую часть дроби приводим к числу как первый числитель
+denominator1 = int(frac_tion1[1])#вторую часть дроби приводим к числу как первый знаменатель
```

Comment:
Но можете сразу поместить необходимое значение в эти переменные, используя функцию `map`.
```suggestion
frac_tion1 = input("Введите первую дробь:")
numerator1, denominator1 = map(int, frac_tion1 .split("/"))  # первую часть дроби приводим к числу как первый 
```